NOTICE.txt

```diff
@@ -3491,3 +3272,167 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 ---
+
+## freetype
+
+This product contains a modified portion of 'freetype', a font rasterizer in the Go programming language, built by Go.
+
+* HOMEPAGE
+  * https://github.com/golang/freetype
+
+* LICENSE
+
+Use of the Freetype-Go software is subject to your choice of exactly one of
+the following two licenses:
+  * The FreeType License, which is similar to the original BSD license with
+    an advertising clause, or
+  * The GNU General Public License (GPL), version 2 or later.
```

Comment:
We cannot use GPL. Propose we keep the original entry of freetype and keep both homepages (.org and github)